src/screens/welcome/Welcome.tsx

```diff
@@ -92,6 +98,22 @@ export const Welcome = ({ route }: AuthNavigationProps<'WELCOME'>) => {
         paddingBottom={isIos ? 'ml' : 'xs'}
         marginBottom="l"
         alignItems="center">
+        <BaseOpacity marginBottom="xl" paddingHorizontal="l" onPress={showPrivacyPolicyModal}>
+          <Text variant="lightGreyRegular" textAlign="left">
+            <Trans
+              t={t}
+              i18nKey="privacyPolicyNormal"
+              components={{
+                b: <Text variant="lightGreyRegularBold" textAlign="left" />,
+              }}
+              values={{ btnLabel: t('seeDemoButton') }}
+            />
+
+            <Text variant="lightGreyRegularBold" textAlign="center" color="primary">
+              {t('privacyPolicyAccent')}
+            </Text>
+          </Text>
+        </BaseOpacity>
```

Comment:
```suggestion
       <ShowPrivacyPolicyButton onShowPrivacyPolicy={showPrivacyPolicyModal} />
```
I'd rather not clutter the welcome screen with translations. Could you move it to a separate component?